config/crd/bases/consul.hashicorp.com_servicedefaults.yaml

```diff
@@ -80,16 +80,133 @@ spec:
                     description: Mode is the mode that should be used for the upstream connection. One of none, local, or remote.
                     type: string
                 type: object
+              mode:
+                description: 'Mode can be one of "direct" or "transparent". "transparent" represents that inbound and outbound application traffic is being captured and redirected through the proxy. This mode does not enable the traffic redirection itself. Instead it signals Consul to configure Envoy as if traffic is already being redirected. "direct" represents that the proxy''s listeners must be dialed directly by the local application and other proxies. Note: This cannot be set using the CRD and should be set using annotations on the services that are part of the mesh.'
```

Comment:
It's weird that it added another single quote here to the `proxy's` and now it's `proxy''s` Don't know if there's a way to escape that.

Reply:
yeah..i poked around and couldnt really find a way to prevent that